jdk/test/java/lang/VirtualThread/YieldQueuing.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @run testng/othervm -Djdk.virtualThreadScheduler.maxPoolSize=1 YieldQueuing
+ * @summary Test Thread.yield submits the virtual thread task to the expected queue
+ */
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.locks.LockSupport;
+
+import org.testng.annotations.Test;
+import static org.testng.Assert.*;
+
+class YieldQueuing {
```

Comment:
It looks the classes and test methods in other tests, including those migrated from Loom, are `public`.
Please refer to [JfrEvents.java].

[JfrEvents.java]:
<https://github.com/Tencent/TencentKona-8/blob/KonaFiber/jdk/test/java/lang/VirtualThread/loom/JfrEvents.java>